client.go

```diff
@@ -196,15 +218,10 @@ func runCommandInContainer(img string, cmd []string, dst io.Writer) (string, err
 // returned can be of type dockerError. This only happens when the container
 // has run normally (no signals, no timeout), but the exit code is not 0. Read
 // the documentation on the `dockerError` command on why we do this.
-func startContainer(img string, cmd []string, wait bool, dst io.Writer) (string, error) {
-	id, err := createContainer(img, cmd)
-	if err != nil {
-		log.Println(err)
-		return "", err
-	}
+func startContainer(id string, cmd []string, wait bool, dst io.Writer) (string, error) {
```

Comment:
Update the documentation as @vrothberg said. The semantics have changed, so the expectation has changed too, and the documentation should be in sync with the semantics.

Reply:
`startContainer(...)` currently doesn't use the `cmd` argument. We need to fix as the interface suggests that passed `cmd` will actually be executed inside container `id`. Sorry for not catching this before.

Reply:
Yeah, `cmd` shouldn't be passed now that we mirror the `create`-`start` API.